src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2403,18 +2445,38 @@ public int getAvcAsInt() {
 			return 0;
 		}
 	}
+	
+	public int getHevcAsInt() {
+		try {
+			return Integer.parseInt(getHevcLevel().replaceAll("\\.", ""));
+		} catch (Exception e) {
+			return 0;
+		}
+	}
 
 	public String getH264Profile() {
 		synchronized (h264ProfileLock) {
 			return h264Profile;
 		}
 	}
+	
+	public String getH265Profile() {
+		synchronized (h265ProfileLock) {
+			return h265Profile;
+		}
+	}
 
 	public void setH264Profile(String s) {
 		synchronized (h264ProfileLock) {
 			h264Profile = s;
 		}
 	}
+	
+	public void setH265Profile(String s) {
+		synchronized (h265ProfileLock) {
```

Comment:
@Sami32 You're not reusing it as it's separate methods. ```s``` only exists in that method's scope, so there's no conflict. 

Using synchronization can however be a bit tricky, as you have to look at the context from which the method is called to make sure you don't create deadlocks.